src/main/resources/jupyter/init-actions.sh

```diff
@@ -102,46 +112,6 @@ if [[ "${ROLE}" == 'Master' ]]; then
     JUPYTER_NOTEBOOK_CONFIG_URI=$(jupyterNotebookConfigUri)
     JUPYTER_NOTEBOOK_FRONTEND_CONFIG_URI=$(jupyterNotebookFrontendConfigUri)
 
-    log 'Installing prerequisites...'
```

Comment:
See PR description. I split our init script in two -- so the "Installing prerequisites" stuff now lives in `init-vm.sh`.